app/main.py

```diff
@@ -2,0 +1,46 @@
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None :
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = False
+        return self.appetite
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None :
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
+    @classmethod
+    def catch_mouse(cls) -> None :
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None :
+        super().__init__(name, appetite=7, is_hungry=is_hungry)
+
+    @classmethod
```

Comment:
The `@classmethod` decorator is not necessary here since `bring_slippers` does not use `cls` or any class-level data. Consider removing the decorator.